builtin/providers/aws/resource_aws_db_instance_test.go

```diff
@@ -49,6 +49,25 @@ func TestAccAWSDBInstance_basic(t *testing.T) {
 	})
 }
 
+func TestAccAWSDBInstance_withoutEngineVersion(t *testing.T) {
```

Comment:
Let's just remove `engine_version` from `testAccAWSDBInstanceConfig`, and then modify this test to just make sure it's present